src/libraries/Common/tests/System/Net/RandomReadWriteSizeStream.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Test.Common
+{
+    public class RandomReadWriteSizeStream: Stream
```

Comment:
Comment here to describe what this is for?

Also nit: space before colon

Reply:
updated.